### PR TITLE
gir fixes

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
As part of https://github.com/gtk-rs/gir/pull/1161, the docs are now based on analysis rather than the symbols list. With that, in order to avoid broken links we must mark missing(not automagically implemented ones) as such to avoid the analysis claiming a function exist and we end up with a broken link.

Do not merge until the gir PR is merged, there are still a bunch of other functions that needs to be handled properly and a ton of broken markdown in gtk3's gir file makes noticing those a bit harder.

Will get back to it in the future. But so far, most of the links are properly working :) no more breakage!

### CODE REVIEW

File: atk/src/attribute.rs [hunk 1]
```diff
@@ -4,6 +4,17 @@ use glib::translate::*;
 use glib::GString;
 use std::fmt;
 
+/// AtkAttribute is a string name/value pair representing a generic
```
Comment (GuillaumeGomez) [inline atk/src/attribute.rs:7]: You'll need to add the rustdoc-stripper annotation, otherwise it'll get removed.

Comment (bilelmoussaoui) [reply atk/src/attribute.rs:7]: I didn't document it myself, it was always there :) I don't plan to fix gtk3, just the broken docs links

Comment (GuillaumeGomez) [reply atk/src/attribute.rs:7]: The diff tells otherwise.

Comment (bilelmoussaoui) [reply atk/src/attribute.rs:7]: probably caused by embed-docs, i will the changes there and regen.

**Comment by sdroege** [comment]
@bilelmoussaoui What's the status here?

**Comment by bilelmoussaoui** [comment]
I don't really plan to rebase this one, if someone wants to take over what I did here; feel free to do so

**Comment by sdroege** [comment]
Something for @GuillaumeGomez I guess

**Pull request closed without merging**